src/components/product-sorting-select/product-sorting-select.module.scss

```diff
@@ -0,0 +1,31 @@
+.select {
+    width: 240px;
+    justify-content: end;
+    background: none;
+    border: none;
+    font: var(--paragraph3);
+}
+
+.selectDropdown {
+    padding: 12px 0;
+    border: 1px solid rgb(23, 17, 13, 0.4);
+    background: var(--primary1);
+    box-shadow:
+        0 4px 8px 0 rgba(0, 0, 0, 0.1),
+        0 0 4px 0 rgba(0, 0, 0, 0.1);
```

Comment:
Lets maybe use `rgba` having 4th param?

Reply:
@tetianaman why? rgba is legacy syntax that is not recommended to use anymore.

Reply:
From spec I understood that comma separated syntax is legacy, not `rgba` function.
But I don't see any difference between modern rgb and rgba syntaxes.

I'll change it to `rgb(23 17 13 / 0.4)`.

Reply:
<img width="799" alt="image" src="https://github.com/user-attachments/assets/14e9511a-64de-497c-85be-72fc6d0436a4">

Reply:
I keep using `rgba()`, becuase Codux color picker doesn't support modern rgb syntax.